lib/index.js

```diff
@@ -21,9 +41,21 @@ bedrock.events.on('bedrock.init', () => {
 
 bedrock.events.on('bedrock.start', () => {
   const {config: {'did-io': cfg}} = bedrock;
-
+  const didKeyDriver = didMethodKey.driver(cfg.methods.key);
```

Comment:
Since passing options to the driver constructor isn't a thing right now, let's remove it in case it might cause unforeseen issues with a minor release:

```suggestion
  const didKeyDriver = didMethodKey.driver();
```

Reply:
Removed [765b3aa](https://github.com/digitalbazaar/bedrock-did-io/pull/16/commits/765b3aa765417b874f47cb8c2c38dbef99e25909)